data.json

```diff
@@ -466,5 +466,11 @@
     "githubUsername": "paulsonjpaul",
     "imageUrl": "https://user-images.githubusercontent.com/73777251/104731775-360a7a80-5762-11eb-8a86-b50d98b58b01.png",
     "issueNumber": 247
-  }
+  },
+  {
+    "name": "Idris Olubisi",
+    "githubUsername": "Olanetsoft",
+    "imageUrl": "https://user-images.githubusercontent.com/45847909/104912483-9da40e00-598c-11eb-8881-971faeaecd9d.png",
+    "issueNumber": 254
+}
```

Comment:
I think this `}` should be lined up like the others.

Reply:
@AllanRegush fixed